submissions/seyifunmiOjo/css/css2/product.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
```

Comment:
You don't have to create a new HTML file for this task. Kindly update the product page you have in your HTML folder. It should be a continuation

Reply:
Done ma. I've put it in its right folder @adaugochi